File: Practice problem/String handling basic.py
# 연습문제 문자열 다루기 기본 Level 1

def solution(s):
    if len(s) != 4 and len(s) != 6:
        return False
    for i in range(len(s)):
        if ord(s[i]) > 57 or ord(s[i]) < 48:
            return False
    return True

File: Practice problem/String handling basic.cpp
// 연습문제 문자열 다루기 기본 Level 1

#include <string>
#include <vector>

using namespace std;

bool solution(string s) {
    if(s.length() != 4 && s.length() !=6){ return false; }
    for(int i = 0; i < s.length(); i++){
        if(s.at(i) > 57 || s.at(i) < 48){ return false; }
    }
    return true;
}
